app.js

```diff
@@ -0,0 +1,27 @@
+const express = require("express")
+const app = express();
+
+const {getTopics} = require("./controllers/topics");
+
+app.get("/api/topics", getTopics);
+
+
+app.use("*", (req, res) => {
+    res.status(404).send({message: "Endpoint not found"})
+})
+
+app.use((err, req, res, next) => {
+    if(err.status){
+        res.status(err.status).send({message: err.message});
+    }else{
+        next();
```

Comment:
Don't really need this error handler at this point. But if you do have it, next should be invoked by err in order to pass it down